pkg/extension/sumologicextension/extension.go

```diff
@@ -90,8 +90,10 @@ var errGRPCNotSupported = fmt.Errorf("gRPC is not supported by sumologicextensio
 var _ configauth.ClientAuthenticator = (*SumologicExtension)(nil)
 
 func newSumologicExtension(conf *Config, logger *zap.Logger) (*SumologicExtension, error) {
-	if conf.Credentials.AccessID == "" || conf.Credentials.AccessKey == "" {
-		return nil, errors.New("access_key and/or access_id not provided")
+	if conf.Credentials.InstallToken == "" {
+		if conf.Credentials.AccessID == "" || conf.Credentials.AccessKey == "" {
+			return nil, errors.New("access credentials not provided: need either install_token or access_id and access_key")
+		}
```

Comment:
Can we add a log here stating that user is using access key+id and they are deprecated? That would encourage people to migrate and give them a clear warning that they should expect this to go away.